packages/spritesheet/src/SpritesheetLoader.ts

```diff
@@ -9,6 +9,20 @@ import type { Loader, ILoaderResource } from '@pixi/loaders';
  *
  * This middleware automatically generates Texture resources.
  *
+ * If you're using Webpack or other bundlers and plan on bundling the atlas' JSON,
+ * use the {@link PIXI.Spritesheet Spritesheet} class to directly parse the JSON.
```

Comment:
@SukantPal I noticed that the `@link` tags here don't work. Is this a known issue in Webdocs

Reply:
@bigtimebuddy Use `{@link PIXI.Spritesheet}`. I think (not sure tho) `[Spritesheet]{@link PIXI.Spritesheet}` might work as well.

Reply:
Here are the JSDoc formats. Is it possible to support the last two (sounds like they aren't supported yet)? I think all these formats are used in our docs.
```
{@link namepathOrURL}
[link text]{@link namepathOrURL}
{@link namepathOrURL|link text}
{@link namepathOrURL link text (after the first space)}
```

Reply:
The first two should be working right now. I'll work on adding the other two.